best-practices/tech_lead.md

```diff
@@ -0,0 +1,318 @@
+# Technical Lead
+
+- "being a good tech lead means some version of: good execution with a team that’s happy." - https://blog.coleadership.com/what-does-it-take-to-be-a-successful-tech-lead/
+- “Success for me in this role is if I can get the team to accomplish the goals that the company is looking for.” - https://blog.coleadership.com/what-does-it-take-to-be-a-successful-tech-lead/
+- "meet with stakeholders outside of the team keeping a good information flow, or to remove blockers."- https://www.patkua.com/blog/the-definition-of-a-tech-lead/
+- "A Tech Lead is a software engineer responsible for leading a team and alignment of the technical direction." - https://www.patkua.com/blog/the-definition-of-a-tech-lead/
+- "Lead will be the interface between the team and the management." - https://www.weetechsolution.com/blog/technical-leader-roles-and-responsibilities
+- "Providing a strong technical direction involves establishing a technical vision, resolving technical disagreements and managing the technical quality of team deliverables. Effective technical leadership ensures the team uses appropriate engineering practices (such as CD or automated testing), invests in continual improvements to tooling or technical debt, and that the system evolves to meet its changing needs and environment."- https://www.patkua.com/blog/the-definition-of-a-tech-lead/
+
+# It's About Team Achievement
+
+"excelling in delivery and execution doesn't just mean being able to do those things yourself, but being able to coordinate, communicate, plan, and delegate effectively." - https://blog.coleadership.com/what-does-it-take-to-be-a-successful-tech-lead/
+
+TL is a full-time job; I don't believe you can be a good TL and a developer at the same time because being TL takes at least 70% of work time, and the remaining time is usually broken up into a few minutes here and there.
+
+## It's about OTHER people
+
+The TL role is to help the _other_ devs be as productive as possible on sprint and work-cycle goals.  The TL is the interface to the info the team needs to work effectively.
+- knock down roadblocks, or help find another way around
+- make sure PRs are reviewed promptly, and with an eye to quality and coding goals (below)
+- ensure team awareness of current priorities, which do shift
+- keep work on track
+- try to anticipate gotchas and address them
+
+The TL may not be the expert in the tech
+- use the team ... tap into expert knowledge wherever it resides
+
+The TL may not be able to code much
+- more devs --> more TL time needed coordinating
+- never take blocking tickets
+  - it's more important for you to be available for pairing, consulting with PO, ticket writing, board grooming, etc.
+- take tiny tickets so they won't languish half done
+- take unfun tickets so other devs can be happier
```

Comment:
This is of course much appreciated, but I am not sure should necessarily be an expectation of the TL.